stock_inventory_discrepancy/security/stock_inventory_discrepancy_security.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Eficent Business and IT Consulting Services S.L.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo noupdate="1">
+
+    <record id="group_stock_inventory_validation" model="res.groups">
+        <field name="name">Validate Inventory Adjustments Under Threshold</field>
+        <field name="category_id" ref="base.module_category_usability"/>
+    </record>
+    <record id="group_stock_inventory_validation_always" model="res.groups">
+        <field name="name">Validate All inventory Adjustments</field>
+        <field name="category_id" ref="base.module_category_usability"/>
+        <field name="implied_ids"
+               eval="[(4, ref('stock_inventory_discrepancy.group_stock_inventory_validation'))]"/>
+        <field name="users" eval="[(4, ref('base.user_root'))]"/>
+    </record>
+
+    <record model="res.groups" id="stock.group_stock_manager">
```

Comment:
Yes, but it may be preferable to keep the usual behavior when installing the module and then manually add users to that group? And the same for manager, just after installing they don't have any extra right, you will have to manually add them to the group that can validate everything. What do you think?